src/fields/Resources/ResourcesItem/ResourcesItem.jsx

```diff
@@ -7,29 +7,38 @@ import dateFormat from '../../../definitions/date-format';
 import './resources-item.scss';
 
 function getFileType(mime, fieldSchema) {
-  const type = fieldSchema.enum.indexOf(mime) > -1 ? fieldSchema.enumNames[fieldSchema.enum.indexOf(mime)].toLowerCase() : 'overig';
+  const type = fieldSchema.enum.indexOf(mime) > -1 ? fieldSchema.enumNames[fieldSchema.enum.indexOf(mime)] : 'Anders';
```

Comment:
Alleen de type verschilt tussen verschillende calls toch? Dan lijkt het me simpeler om de type terug te geven en in de hoofd methode de React render code te zetten. Een functie die een string teruggeeft is een stuk beter te bevatten dan een React element.